### DISCUSSION TITLE
Other support

### DISCUSSION THREAD

**Discussion by lewisl** [kagisearch/kagi-docs#338] [Kagi Search API Feature Requests & Ideas]
I would be willing to make a small donation to support the Orion effort, but I see no benefit to Orion+.  Indeed, getting more frequent beta downloads is a disadvantage. Also, if I have a valid bug report or meaningful suggestion, I assume you'll eventually give it a look see and do as you see fit. I don't need you to acknowledge whether you looked or indicate any decision you made.  You're in charge!  How it is possible to make a small donation to the effort?

**Comment by vprelovac** [comment]
You can chose to 'sponsor issue' on orionfeedback.org which will allow you to donate via Paypal.

**Comment by lewisl** [comment]
Thanks!

From: Vladimir Prelovac ***@***.***>
Reply-To: kagisearch/kagi-docs ***@***.***>
Date: Sunday, February 4, 2024 at 5:25 PM
To: kagisearch/kagi-docs ***@***.***>
Cc: Lewis Levin ***@***.***>, Author ***@***.***>
Subject: Re: [kagisearch/kagi-docs] Other support (Discussion #338)


You can chose to 'sponsor issue' on orionfeedback.org which will allow you to donate via Paypal.
